create_package.py

```diff
@@ -119,20 +119,17 @@ def parse_args():
 
 
 def update_yml_manifest(root_path, v_major, v_minor, v_revision, v_build, rh_target):
-    manifest_template = Path(root_path, "manifest_rh6.template" if rh_target == '6' else "manifest_rh7.template")
+    manifest_template = Path(root_path, "manifest_rh7.template")
```

Comment:
Do we use the rh7 template also for rh8?

Reply:
Yep it's the same. I could rename it.